docs/modules/ups_openshift/index.asciidoc

```diff
@@ -0,0 +1,56 @@
+// ---
+// layout: post
+// title: Running the UnifiedPush Server on OpenShift
+// section: guides
+// ---
+//
+
+[[openshift]]
+== Running the UnifiedPush Server on OpenShift
+
+AeroGear provides a link:https://github.com/aerogearcatalog/unifiedpush-apb[Ansible Playbook Bundle] to make it easy to get running on OpenShift.
+If you don't already have an account you can sign up link:https://www.openshift.com/pricing/index.html[here]. You can also use one of the various ways to run OpenShift on your own hardware like link:https://github.com/minishift/minishift[Minishift], link:https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md#getting-started[oc cluster up] or the link:https://github.com/aerogear/mobile-core[Mobile Application Platform].
```

Comment:
Maybe we could leave minishift & openshift as the only options,
then in a separate sentence link off to the mobile-core repo saying something like `Alternatively, you can follow instructions in the link:https://github.com/aerogear/mobile-core[Mobile Core] repo to get a local cluster running`